File: 3. Arrays/1. Easy/C++/1. Largest element in an array.cpp
/*
Find largest element in an array
*/

#include <bits/stdc++.h>
using namespace std;


int findLargestElement(int arr[],int n)
{
    int largest = arr[0];
    for(int i=0;i<n;i++){
        if(arr[i]>largest){
            largest = arr[i];
        }
    }
    return largest;
}

int main() {
	// your code goes here
    int n;
    cin >> n;
    int arr[n];
    for(int i=0;i<n;i++){
        cin >> arr[i];
    }
    int largest = findLargestElement(arr,n);
    cout << largest;
	return 0;
}

File: 3. Arrays/1. Easy/Python/1. Largest element in an array.py
# cook your dish here
#Find largest element in an array


def findLargest(arr,n):
    largest = arr[0]
    for i in range (0,n):
        if(arr[i]>largest):
            largest = arr[i]
    return largest

n = int(input())
arr = list(map(int,input().strip().split()))
largest = findLargest(arr,n)
print(largest)